app/controllers/stories_controller.rb

```diff
@@ -11,6 +11,8 @@ class StoriesController < ApplicationController
     ]
   }.freeze
 
+  SIGNED_OUT_RECORD_COUNT = (Rails.env.production? ? 60 : 10).freeze
```

Comment:
Obligatory generalization question: will every community want the same values, or do we need to account for configurability here?

Reply:
I think they likely will. I feel like this is mostly a technical detail about how we load a page. Not something an admin should care about.

If the value changes, I think it would be maybe based on an automated condition (e.g. if the site doesn't have enough content to fulfill some pagination expectation), but nothing the admin should be concerned with.

Reply:
👍 thanks for the explanation @benhalpern